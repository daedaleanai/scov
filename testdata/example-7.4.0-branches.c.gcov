file:example.c
function:28,1,main
lcount:28,1
lcount:34,1
branch:34,taken
branch:34,nottaken
lcount:36,1
lcount:37,1
lcount:43,1
lcount:44,1
lcount:49,1
branch:49,nottaken
branch:49,taken
lcount:51,0
lcount:55,1
lcount:58,1
