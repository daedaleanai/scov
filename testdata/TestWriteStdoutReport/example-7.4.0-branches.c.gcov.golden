  Line coverage: [##################  ]  90.0%  (9/10)
  Func coverage: [####################] 100.0%  (1/1)
Branch coverage: [##########          ]  50.0%  (2/4)
Region coverage:  No data
